packages/calcite-components/src/components/modal/modal.e2e.ts

```diff
@@ -520,10 +520,10 @@ describe("calcite-modal accessibility checks", () => {
     const modal = await page.find("calcite-modal");
     await modal.setProperty("open", true);
     await page.waitForChanges();
-    const documentClass = await page.evaluate(() => {
-      return document.documentElement.classList.contains("overflow-hidden");
+    const isOverflowHIdden = await page.evaluate(() => {
```

Comment:
Typo: `isOverflowHIdden` (uppercase `I` in `Hidden`)

Applies to similar util used below.